lua/autorun/server/sv_votestuff.lua

```diff
@@ -0,0 +1,53 @@
+if !SERVER then return end
+
+util.AddNetworkString( "tv::SendTasks" )
+util.AddNetworkString( "tv::SendAnswer" )
+
+Seconds = { // Each X seconds will appear a vote for all online players. They will have X/2 seconds to anwser;
+	10,
```

Comment:
So you have logic lower down for selecting a random number from this table, but there is only one value in the table. This code is obviously unfinished and i'm not interested in finishing it for you.

Also "Seconds" is an inappropriate name for a global variable; it is likely to conflict with something else with equivalently careless variable naming.